app/src/main/res/values/strings.xml

```diff
@@ -227,6 +227,10 @@
 
     <!-- Error handling -->
     <string name="retry">Retry</string>
+
+    <!-- Map Utils -->
+    <string name="facility_marker">facility_marker</string>
```

Comment:
Nitpick: Since this string never really be translated, it is imo not really necessary to save is as a string resource, its enough to just save it as a component in some object, like Facility or so. But I may be wrong, this side of Kotlin is not my strength.

Reply:
Done